src/status_im/ui/screens/browser/views.cljs

```diff
@@ -0,0 +1,99 @@
+(ns status-im.ui.screens.browser.views
+  (:require-macros [status-im.utils.views :as views])
+  (:require [re-frame.core :as re-frame]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.screens.browser.styles :as styles]
+            [status-im.ui.components.status-bar.view :as status-bar.view]
```

Comment:
Usual convention is `[status-im.ui.components.status-bar.view :as status-bar]` and  `[status-im.ui.components.status-bar.styles :as status-bar.styles]`

Reply:
didn't know that

Reply:
That's empirical but it's usually used.